src/main/java/org/broadinstitute/hellbender/utils/read/ArtificialSAMUtils.java

```diff
@@ -0,0 +1,267 @@
+package org.broadinstitute.hellbender.utils.read;
```

Comment:
Please delete this `ArtificialSAMUtils class completely! (and replace with the hellbender-native `ArtificialReadUtils`)